### ISSUE TITLE
[Android app] Old phone can't upgrade Wire app to latest version; backup file can't be restored on new phone. Please help!

### ISSUE THREAD

**Issue by JamesDBartlett3** [wireapp/wire#411]
On my old Android phone, the Wire app has been stuck on version 3.82.38 for a long time. The Play Store just refuses to upgrade it, and doesn't give a reason. It just says: 
>Can't install Wire - Secure Messenger. Try again, and if it still doesn't work, [see common ways to fix the problem](https://support.google.com/googleplay/answer/14122894?hl=en&error=6004&ctx=com.wire&isid=mQsOF39EQSu78akS81cQtA&visit_id=638545996928539845-1575613289&p=fix_download_install&rd=1).  

I've already tried all of those "common ways to fix the problem" multiple times, and nothing helped. But the old version of the Wire app still works just fine, I'm still logged in, I can still send and receive messages, and my messages still sync properly across all of my devices.

However, I recently got a new Android phone, so I did what I've always done in the past: I created a Wire backup on my old phone, transferred the backup file to my new phone, installed the latest version of the Wire app on the new phone, and initiated a restore operation from the backup file. But the Wire app on the new phone refused to restore from backup, displaying this error message:
>Incompatible backup. This backup was created by a newer or outdated version of Wire and cannot be restored here.

So I thought I'd try something clever. I downloaded and installed the same old version of the Wire app that's installed on my old phone, thinking I would restore the backup, then upgrade the Wire app to the latest version. But the older version of the Wire app won't let me login to my account, so I can't restore the backup that way either.

TL;DR: I can't restore my backup because it was made by an old version of the Wire app, but I can't upgrade the Wire app to a later version to make a compatible backup file, and I can't login on a version of the app that is compatible with the backup file and upgrade the app after restoring the backup, so what am I supposed to do here?

**Comment by MohamadJaara** [comment]
Hi @JamesDBartlett3 
sorry to hear about your issue, the minimum android version supported for the new app is android 7. 
And the backup between the old and the new apps are incompatible, as a solution, I can recommend installing the old app on the new device, login and restoring the backup and then updating the app from the play store

If you require any further assistance or suggestions, please feel free to reach out to us. We are always happy to help.

**Comment by JamesDBartlett3** [comment]
> as a solution, I can recommend installing the old app on the new device, login and restoring the backup and then updating the app from the play store

@MohamadJaara, as I mentioned in my original comment, I already tried that, and it didn't work:

> I downloaded and installed the same old version of the Wire app that's installed on my old phone, thinking I would restore the backup, then upgrade the Wire app to the latest version. But the older version of the Wire app won't let me login to my account, so I can't restore the backup that way either.

**Comment by MohamadJaara** [comment]
Hi, I apologize for the confusion and for any inconvenience this situation may have caused. Today, I was experimenting with this issue and managed to log in using an older version of Scala with this [APK](https://drive.google.com/file/d/108Xq7RVsjXmYiek_hN9LJa0pxHHxCSoI/view?usp=sharing). It’s an old production version of the app. If you still have the old backup, please follow these steps to log in:

1. Install the APK linked above.
2. Before opening the app, turn off your internet connection (this will prevent the app update prompt).
3. Open the app.
4. Now, enable the internet and log in (be sure to restore the backup when prompted).
5. Update the app to the [latest release version](https://github.com/wireapp/wire-android/releases/tag/v4.8.0).
6. The new version will start migrating your history. Keep the app open and let it complete the process.
7. Once done, create a new backup in the updated format.

If you encounter any further issues, please don’t hesitate to report them. We’re happy to help.

**Comment by MohamadJaara** [comment]
Unfortunately, the old, repo is not accusable, and I had to dig an outdated version from my archive

**Comment by JamesDBartlett3** [comment]
@MohamadJaara thank you for looking into this for me. I just downloaded the APK files you suggested, and now I will follow the instructions you provided, and see if I can get it working.

**Comment by JamesDBartlett3** [comment]
@MohamadJaara, that worked! Thank you so much for your assistance with this! 😊
